### PR TITLE
mon/Monitor: add a newline to 'ceph health [detail] -f [json|xml]' output

### PR DESCRIPTION
**Pull request by shminjs**
add a newline to 'ceph health [detail] -f [json|xml]' output to keep up with other command

Signed-off-by: shimin <shminjs@outlook.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/mon/Monitor.cc [hunk 1]
```diff
@@ -3607,13 +3607,12 @@ void Monitor::handle_command(MonOpRequestRef op)
       }
       rdata.append(ds);
     } else if (prefix == "health") {
-      string plain;
-      healthmon()->get_health_status(detail == "detail", f.get(), f ? nullptr : &plain);
+      healthmon()->get_health_status(detail == "detail", f.get(), f ? nullptr : &ds);
```
Comment (tchaikov) [inline src/mon/Monitor.cc:3610]: this does not compile. `ds` is a `stringstream`, while this parameter is supposed to be a `string*`.

**Review by tchaikov — CHANGES_REQUESTED** [review]

File: src/mon/Monitor.cc (hunk 1 above)
Comment (shminjs) [reply src/mon/Monitor.cc]: ops, I should compile first.

**Comment by tchaikov** [comment]
@shminjs ping?

**Comment by shminjs** [comment]
> @shminjs ping?

I will close it.

**Comment by tchaikov** [comment]
@shminjs i am not sure why you plan to close this PR. what i suggest is to remove "ds << '\n';". the change makes sense in general.

**Comment by shminjs** [comment]
> @shminjs i am not sure why you plan to close this PR. what i suggest is to remove "ds << '\n';". the change makes sense in general.

I add this line "ds << '\n'" to make command output in same style. It is a very minor style problem.

**Comment by tchaikov** [comment]
if you take a closer look at the implementation of Formatter, you probably can find out that Formatter add the `\n` with the `flush()` variant you are using. 

anyway, thanks for your contribution. i will create another PR to address this issue.

**Comment by shminjs** [comment]
> if you take a closer look at the implementation of `buffer::list`, you probably can find out that `buffer::list::append()` adds the `\n` already.
> 
> anyway, thanks for your contribution. i will create another PR to address this issue.

😂I figure out now, thx.

**Pull request closed without merging**